pkg/git/git_test.go

```diff
@@ -83,6 +86,13 @@ func TestRepo_BestRefFor(t *testing.T) {
 			release: ReleaseRef,
 			rule:    AnyRule,
 		},
+		"Any - v0.2 - with submodule": {
+			repo:    submoduleRepo,
+			version: semver.MustParse("0.2.0"),
+			want:    "ref@release-0.2",
```

Comment:
I think this should be `ref@sub/release-0.2` no? is this not a tag? perhaps this is a branch and I don't fully understand what the goal is.

Reply:
We thought to go for the branches for submodules. Otherwise, we would need to add additional Git tags to the repo like: `sub/v0.2.0`